lib/install.js

```diff
@@ -93,8 +93,14 @@ exports.install = function(targets, options) {
     if (options.link || options.quick)
       options.lock = true;
 
+    var defaultTargets = options.dev ? config.pjson.devDependencies : config.pjson.dependencies;
     if (targets === true)
-      targets = options.dev ? config.pjson.devDependencies : config.pjson.dependencies;
+      targets = defaultTargets;
+    else if (options.doUpdate)
+      Object.keys(defaultTargets).forEach(function(module) {
+        if (module in targets)
```

Comment:
@crisptrutski I think we should probably just ignore versions for the update command. Perhaps the right behaviour would be to throw.